lib/Data/Validate/Sanctions/Redis.pm

```diff
@@ -0,0 +1,230 @@
+package Data::Validate::Sanctions::Redis;
```

Comment:
I would suggest naming it `Data::Validate::Sanctions::Storage` instead of `Data::Validate::Sanctions::Redis`. `Redis` is just a system that we are using, we may change this in future

Reply:
This module is exclusively created for redis storage. It won't work without `hmget` and `hmset` calls. A better alternative IMHO is to use the parent class `Data::Validate::Sactions` as a **Factory** that can create object of this subclass with a code like this:
```
my $validator = Data::Validate:::Sanctions->new(storage => 'redis`, connection => $redis);
```